tools/underlay.repos

```diff
@@ -11,10 +11,6 @@ repositories:
     type: git
     url: https://github.com/ros-simulation/gazebo_ros_pkgs.git
     version: ros2
```

Comment:
I believe we need image transport for WP follower, if that's not the case, please remove the header include / package.xml find / cmakelists find so we can fully remove it from our dependency tree

Reply:
From commit log, I've mentioned that's it is now included in the ros2.repos file, so comes bundled with the ros2 master image we build from. So no real need to rebuild it in the underlay.